.github/workflows/build.yml

```
@@ -10,76 +10,164 @@ on:
     outputs:
       base_tag: ${{ steps.set-tag.outputs.base_tag }}
+      sha_tag: ${{ steps.set-tag.outputs.sha_tag }}
+      cache_tag: ${{ steps.set-tag.outputs.cache_tag }}
     steps:
-      - name: Set BASE_IDE_TAG
+      - name: Checkout for SHA access
+        uses: actions/checkout@v4
+
+      - name: Set tags
         id: set-tag
         run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2129:style:18:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)